website/source/guides/integrations/consul-connect/index.html.md

```diff
@@ -262,8 +262,11 @@ dashes (`-`) are converted to underscores (`_`) in environment variables so
  - Consul Connect Native is not yet supported.
  - Consul Connect HTTP and gRPC checks are not yet supported.
  - Consul ACLs are not yet supported.
- - Only the Docker, exec, and raw exec drivers support network namespaces and
+ - Only the Docker and raw exec drivers support network namespaces and
    Connect.
+ - The exec driver supports network namespaces but doesn't currently
+   support `bridge` networking. It will support Connect once Connect Native
```

Comment:
I don't think the last part of this is true. We don't need Connect Native (which I'm assuming means supporting the native option in connect) to run this on exec.

Reply:
Updated, along with the note about `java` now that we'll have https://github.com/hashicorp/nomad/pull/6368